### PR TITLE
KLBP-422: Added file validation for doc with max 10mb.

### PR DESCRIPTION
**Pull request by iuliudascalu**

### CODE REVIEW

File: src/enums/common.ts [hunk 1]
```diff
@@ -39,6 +39,10 @@ export const ALERT_MESSAGES: IAlertMessage = {
     type: 'error',
     message: 'La taille du fichier dépasse la limite autorisée de 20 Mo.',
   },
+  fileSize10Mb: {
+    type: 'error',
+    message: 'La taille du fichier dépasse la limite autorisée de 10 Mo.',
+  },
```
Comment (mariapitechplus) [inline src/enums/common.ts:45]: Please add a test for this new validation

**Review by ioan-iacob-netea — APPROVED** [review]

**Review by mariuspop86 — APPROVED** [review]

File: src/enums/common.ts (hunk 1 above)
Comment (iuliudascalu) [reply src/enums/common.ts:45]: Done.

File: src/__mock__/documentMock.ts [hunk 2]
```diff
@@ -17,3 +17,18 @@ export const firstUploadedDocument: IDocument = {
   status: 'pending',
   needsVerso: false,
 }
+
+/**
+ * Size is in Bytes
+ * @param size
+ */
+export function generateFileWithSize(size: number): File {
```
Comment (mariuspop86) [inline src/__mock__/documentMock.ts:25]: There is a file utils_test, move this function there

Comment (iuliudascalu) [reply src/__mock__/documentMock.ts]: Done.

**Pull request merged**